tests/unit/routes/not-found-test.js

```diff
@@ -0,0 +1,12 @@
+import { setupTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+module('Unit | Route | not-found', function (hooks) {
+  setupTest(hooks);
+
+  test('it exists', function (assert) {
+    let route = this.owner.lookup('route:not-found');
```

Comment:
Just for consistency with other route test files.

```suggestion
    const route = this.owner.lookup('route:not-found');
```